data.py

```diff
@@ -2,30 +2,34 @@
 import os.path
 from dotenv import load_dotenv
 
-#Primarily handles user info
-class Data():
+
+# Primarily handles user info
+class Data:
     def __init__(self):
-        self.key_filename = 'eleven_labs_keys' + '.pkl'
-    
+        self.key_filename = "eleven_labs_keys" + ".pkl"
+
     def open_pickle(self):
         if os.path.isfile(self.key_filename):
-            with open(self.key_filename, 'rb') as file:
+            with open(self.key_filename, "rb") as file:
                 return pickle.load(file)
         else:
             return {}
-        
+
     def save_eleven_labs_key(self, user_id, key):
         eleven_labs_key_dict = self.open_pickle()
 
         eleven_labs_key_dict[user_id] = key
 
-        with open(self.key_filename, 'wb') as file:
+        with open(self.key_filename, "wb") as file:
             pickle.dump(eleven_labs_key_dict, file)
 
     """
-    If user is registered, returns the tuple (True, <key associated with the user>)
-    If user is  not registered, returns the tuple (False, None)
+    If user is registered, returns the tuple:
+        (True, <key associated with the user>)
+    If user is not registered, returns the tuple:
+        (False, None)
```

Comment:
[`Typing`](https://docs.python.org/3/library/typing.html) might be nice for this, as well as for all the other files and whatnot

Reply:
Agreed